int32 value
---
int32 errorCode
string errorDescription